### DISCUSSION TITLE
how will you respond to updates made by automatic1111´s main webui repo?

### DISCUSSION THREAD

**Discussion by IkariDevGIT** [vladmandic/sdnext#624] [Q&A]
will you update the same stuff or will it branch off completely and you do your own stuff? @vladmandic

**Comment by vladmandic** [comment]
my plan is that anything that is relevant should be ported here as well. yes, its a non-trivial task, but should be done.

**Reply by IkariDevGIT** [reply]
so you will stay completely updated with the main repo? and what if something is not compatible, will you search for a different "implementation" or will you let it be(not update this repo with the feature)?

**Reply by vladmandic** [reply]
any "not compatible" case would have to investigated on per-case basis. in 99% of cases, it can be made compatible.

**Reply by IkariDevGIT** [reply]
thank you so much for the fast reply, i was worried for a bit because auto1111 is going like a rocket like in the early SD webui times on the dev branch. Will you be able to keep up?

**Reply by vladmandic** [reply]
don't know, will have to try :)

**Reply by IkariDevGIT** [reply]
i hope so, i really like this fork! Are you the only one who frequently works on this?

**Reply by vladmandic** [reply]
on the main branch, right now yes (i hope not for long), there are multiple owners for different feature-branches.

**Reply by IkariDevGIT** [reply]
I hope it doesn't happen like with auto1111 who gets burned out because no one is helping him! Do you have some "strategy" on how to solve this?

**Reply by vladmandic** [reply]
i do, but it depends on others to buy-in on that :)
i hope to have some updates soon...

**Reply by IkariDevGIT** [reply]
Thank you so much, you are the best! Ande one last question: will you update this fork with the originals master branch only or with the dev branch too?

**Reply by vladmandic** [reply]
right now its run-before-walking (so commit direct to master) - get as much as done as possible to catch-up on items i really want to do. and once that is done (i'd assume ~2 weeks), definitely use dev branch from that point onwards.

**Reply by IkariDevGIT** [reply]
btw send to controlnet does not work on img browser

**Reply by vladmandic** [reply]
If it works elsewhere, it should be reported to image browser extension repo.

**Comment by myndxero** [comment]
Awesome mini Q&A here, maybe worth a pin if that's a thing on github? Trying to fully catch up on relevant things during my absent time.
